internal/pkg/consensus/expected.go

```diff
@@ -324,21 +322,11 @@ func verifyBLSMessageAggregate(sig types.Signature, msgs []*types.UnsignedMessag
 	return nil
 }
 
-func combineMessages(blsMessages [][]*types.UnsignedMessage, secpMessages [][]*types.SignedMessage) [][]*types.SignedMessage {
-	messages := [][]*types.SignedMessage{}
-	for _, msgs := range blsMessages {
-		messages = append(messages, wrapMessages(msgs))
-	}
-	for _, msgs := range secpMessages {
-		messages = append(messages, msgs)
-	}
-	return messages
-}
-
-func wrapMessages(blsMessages []*types.UnsignedMessage) []*types.SignedMessage {
-	signed := []*types.SignedMessage{}
-	for _, msg := range blsMessages {
-		signed = append(signed, &types.SignedMessage{Message: *msg})
+// Unwraps nested slices of signed messages.
+func unwrap2(smsgs [][]*types.SignedMessage) [][]*types.UnsignedMessage {
```

Comment:
Why the 2 in `unwrap2`?

Reply:
For the nested slices (originally types.Unwrap was here too). Removed.